Kentor.AuthServices/SAML2P/Saml2Response.cs

```diff
@@ -369,6 +369,7 @@ private void ValidateSignature(IOptions options)
         /// <summary>Checks the signature.</summary>
         /// <param name="signedRootElement">The signed root element.</param>
         /// <param name="idpKey">The assymetric key of the algorithm.</param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage( "Microsoft.Naming", "CA2204:Literals should be spelled correctly", MessageId = "AuthServices" )]
```

Comment:
Adding AuthServices to CustomDictionary.xml creates dozens of code analysis errors like "CA1702 Compound words should be cased correctly. The compound word 'AuthServices' in namespace name 'Kentor.AuthServices.Metadata' exists as a discrete term. If your usage is intended to be single word, case it as 'Authservices'. Kentor.AuthServices	(Global)"